mods/ra/maps/soviet-05/rules.yaml

```diff
@@ -84,7 +84,14 @@ APC:
 
 DOME:
 	ExternalCondition@lua:
-		Condition: captured
+		Condition: french
+	WithColoredOverlay@IDISABLE:
+		RequiresCondition: !french && disabled
+	Targetable:
+		RequiresCondition: !french
+	Targetable@NOAUTOTARGET:
+		RequiresCondition: french
+		TargetTypes: Ground, Structure, C4, DetonateAttack, SpyInfiltrate, NoAutoTarget
```

Comment:
The targetable traits can be removed now. This is not needed anymore since the current release doesn't automatically  target buildings.

Reply:
I think it is better to keep Radar non-auto targeted even on Attack-Anything stance here, which is what this does.

Reply:
I understand you. The only thing is, this is the only building with such a rule in the campaign/skirmish.

Reply:
We might want to change the other buildings as well then.

Reply:
Wouldn't that be just an error of the player self by choosing attack anything while there is an important building? Just a thought, I won't object it.

Reply:
I agree with @Arular101 here: so long as we we properly and promptly fail the mission (as opposed to leaving it in a state where the game continues but is unwinnable) then it really falls on the player to not put their units into the non-default state where they will fail the mission for you.